public/app/features/explore/ExplorePage.test.tsx

```diff
@@ -100,39 +133,50 @@ describe('ExplorePage', () => {
       });
     });
 
+    // TODO: the following tests are using the compact format, we should use the current format instead
```

Comment:
Could you create a task in the backlog for it and cross reference it here?

Reply:
this is gonna be refactored anyway in the next PR that introduces migrations as each one of them will have to be tested against their specific url format.